pkgs/development/python-modules/setuptools-scm/setup-hook.sh

```diff
@@ -0,0 +1,32 @@
+# Let built package know its version.
```

Comment:
reposting here to not leave the comments in the closed PR:
Should we clean up SETUPTOOLS_SCM_PRETEND_VERSION now?

Reply:
Working on that, will push a treewide commit in a few minutes.